introduction.md

```diff
@@ -4,4 +4,9 @@ Here is more stuff with things so the file is big enough to register for the ass
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?